src/Umbraco.Core/Persistence/Repositories/DataTypeDefinitionRepository.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
Okay, well if it was removed in a previous commit within the PR then that could be why :)

If your previous commits no longer add anything to the PR then it might be an idea to squash your commits.